.travis.yml

```diff
@@ -23,32 +26,32 @@ matrix:
      python: 2.7
    - env: TOXENV=py27-dj110-wt110
      python: 2.7
+   - env: TOXENV=py27-dj110-wt111
+     python: 2.7
+   - env: TOXENV=py27-dj111-wt17
+     python: 2.7
+   - env: TOXENV=py27-dj111-wt18
```

Comment:
This version combination is not supported by Wagtail (https://github.com/wagtail/wagtail/blob/a9b7265784761d7bd9428b9721bfea9486370c30/docs/releases/upgrading.rst).